ProjectProposal.md

```diff
@@ -0,0 +1,27 @@
+# What are we building
+We are building a simple web-app that allows users to maintain an online task list. 
+
+## Main features
+- Create, read, update, and delete tasks from the web interface
+- Tasks have a title and a description
+- Tasks can be marked as new, active, or completed
+- Tasks are stored in the cloud and can be accessed via a user account on any device
+
+## Stretch goals
+- Mobile interface
+- Due dates for tasks
+- Reminders
+- Organize tasks into groups and hierarchies
+- Mark certain tasks as important
+
```

Comment:
Very great job @listonjesse . Shall we add a search bar by keyword in the web app. Thank you so much for your work

Reply:
Feel free to add that to the stretch goals